scripts/globals/job_utils/dragoon.lua

```diff
@@ -75,7 +75,13 @@ end
 
 -- Generic Function for damage-based Jumps
 local function performWSJump(player, target, action, params)
-    local taChar = player:getTrickAttackChar(target)
+    local taChar = nil
+
+    -- Check if preformer is PC to avoid (HIT DEBUG CONDITION: m_PBaseEntity->objtype != TYPE_PC) in lua_baseentitity.
```

Comment:
Going to fix this in core, as I'm not a huge fan of breaking instead of sanely logging.